.github/workflows/haskell.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         key: ${{ runner.OS }}-${{ matrix.ghc }}-cabal-0
 
     - name: Run ShellCheck
-      uses: ludeeus/action-shellcheck@master
+      uses: ludeeus/action-shellcheck@2f2aa0d
```

Comment:
What does this do?

Reply:
Workaround for [this](https://github.com/ludeeus/action-shellcheck/issues/21)